src/pvrclient-mediaportal.cpp

```diff
@@ -2481,3 +2482,52 @@ PVR_ERROR cPVRClientMediaPortal::GetStreamReadChunkSize(int& chunksize)
   chunksize = 32 * 1024;
   return PVR_ERROR_NO_ERROR;
 }
+PVR_ERROR cPVRClientMediaPortal::GetRecordingEdl(const kodi::addon::PVRRecording& recording, std::vector<kodi::addon::PVREDLEntry>& edl) // bilkusg
+// liberally adapted from pvr.vuplus ( recordings.cpp in particular)
+{
+    cRecording* myrecording = GetRecordingInfo(recording);
+    const char* pvrFilePath = myrecording->FilePath();
+    const std::string& pvrSmbPath = m_tsreader->TranslatePath(pvrFilePath);
+    const std::string edlFile = pvrSmbPath.substr(0, pvrSmbPath.find_last_of('.')) + ".edl";
+    kodi::Log(ADDON_LOG_INFO, "GetRecordingEdl for %s from %s", pvrFilePath,edlFile.c_str());
+    FileReader *fileReader = new FileReader();
+    long fileOpenResult = fileReader->OpenFile(edlFile);
```

Comment:
This assumes that the edl always exists? Maybe better to check first if the .edl indeed exist and return early if it does not exist...

Reply:
Yes, makes sense. That could even go the ReadFileContents functions.